composer.json

```diff
@@ -4,9 +4,43 @@
     "license": "GPL-2.0",
     "minimum-stability": "stable",
     "require": {
+        "ezsystems/platform-ui-bundle": "dev-proto_hybrid_package as 2.0.x-dev",
+        "ezsystems/ezpublish-kernel": " ^6.9@dev",
+        "symfony/symfony": "~2.8 | ~3.0",
+        "friendsofsymfony/jsrouting-bundle": "^1.6",
+        "ezsystems/hybrid-platform-ui-assets": "^0.1",
+        "composer/installers": "dev-ezplatform_assets as 1.3.x-dev"
     },
     "require-dev": {
+        "phpspec/phpspec": "^3.2",
+        "memio/spec-gen": "^0.6"
     },
     "autoload": {
-    }
+        "psr-4": {
+            "EzSystems\\HybridPlatformUiBundle\\": "src/bundle",
+            "EzSystems\\HybridPlatformUi\\": "src/lib",
+            "specs\\EzSystems\\HybridPlatformUi\\": "specs"
+        }
+    },
+    "config": {
+        "bin-dir": "bin"
+    },
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/dpobel/ez-field-edit.git"
```

Comment:
let's remove those repositories (as stated above, only https://github.com/dpobel/installers.git is needed)